python/lsst/meas/deblender/deblend.py

```diff
@@ -467,67 +470,67 @@ class MultibandDeblendConfig(pexConfig.Config):
                               doc=("Maximum number of iterations to deblend a single parent"))
     relativeError = pexConfig.Field(dtype=float, default=1e-3,
                                     doc=("Relative error to use when determining stopping criteria"))
-    absoluteError = pexConfig.Field(dtype=float, default=0,
-                                    doc=("Absolute error to use when determining stopping criteria"))
-
-    # Position Fitting Criteria
-    fitPositions = pexConfig.Field(dtype=bool, default=True,
-                                   doc=("Whether or not to fit the positions for each source"
-                                        "while deblending"))
-    diffImgShift = pexConfig.Field(dtype=float, default=0.1,
-                                   doc=("Amount to shift the differential image during a position update."
-                                        "This field is ignored if fitPositions is False."))
-    minTranslation = pexConfig.Field(dtype=float, default=1e-8,
+
+    # Blend Configuration options
+    minTranslation = pexConfig.Field(dtype=float, default=1e-3,
```

Comment:
Why the enormous difference in value?

Reply:
Test revealed that there was no need to waste CPU cycles on higher precision positions.